### ISSUE TITLE
Missing engines Field Update for Node.js Version Support in v2.0.0

### ISSUE THREAD

**Issue by Phillip9587** [pillarjs/finalhandler#64]
Hi `finalhandler` maintainers,

I noticed in the changelog for the latest release (v2.0.0) that there is a breaking change which states:

> drop support for node <18

However, it seems that the `package.json` `engines` field was not updated to reflect this change.

Would it be possible to update the `engines` field in `package.json` to explicitly specify the range of supported Node.js versions? This would greatly help in ensuring compatibility when upgrading to `finalhandler` v2.0.0.

Thank you for all your hard work on this project!

**Comment by wesleytodd** [comment]
Yeah we can do this. We decided to add it as a minor in a few other packages because it was missed in the initial release. While it is technically a breaking change, it's omission was an oversight and so aligns with the policy we have set.

**Comment by Phillip9587** [comment]
There already is PR #59 for this change.

**Comment by wesleytodd** [comment]
Oh cool, I forgot about that one. Lets close this then and use that for any further discussion.
